source/backend/api/Services/CompensationRequisitionService.cs

```diff
@@ -128,5 +125,15 @@ private void AddNoteIfStatusChanged(long compensationRequisitionId, long acquisi
 
             _entityNoteRepository.Add(fileNoteInstance);
         }
+
+        private void CheckDraftStatusUpdateAuthorized((bool? currentStatus, bool? newStatus) statusComparable)
+        {
+            if (statusComparable.currentStatus.HasValue && statusComparable.currentStatus.Value.Equals(false)
+                && ((statusComparable.newStatus.HasValue && statusComparable.newStatus.Value.Equals(true)) || !statusComparable.newStatus.HasValue)
+                && !_user.HasPermission(Permissions.AdminUsers))
+            {
+                throw new NotAuthorizedException();
+            }
+        }
```

Comment:
I see here that we are checking for this permission: **AdminUsers** - which according to the code comments "Can administer user accounts." 

This is not the same as an ADMIN user which would have the `Permissions.SystemAdmin` claim - wouldn't it make sense to use SystemAdmin here instead?

Reply:
that naming convention for claims is certainly ambiguous. If that's the one needed I'll change it.

Reply:
Double check with @devinleighsmith but I believe SystemAdmin is the correct one...

Reply:
updated